docs/node.md

```diff
@@ -27,32 +27,39 @@ it will compile ES6 code before running it.
 
 Launch a REPL (Read-Eval-Print-Loop).
 
+> You should install `@babel/node` and `@babel/core` first before `npx babel-node`, otherwise `npx` will install out-of-dated legacy `babel-node` 6.x.
+
+```sh
+npx babel-node
+```
+
+Install basic dependencies on-the-fly and launch a REPL
 ```sh
-npx @babel/node
+npx --package=@babel/{core,node,preset-env} babel-node
```

Comment:
This won't use `@babel/preset-env`, right? If it doesn't, it should be removed to avoid confusion

Reply:
Also, I find this more clear, since not everyone knows about `{ }` (and I don't know if they work in every shell)

```
npx -p @babel/core -p @babel/node babel-node
```

Reply:
> If it doesn't, it should be removed to avoid confusion

True, we still need a babel config to use `@babel/preset-env`.

> I find [expanding brace expressions] more clear

True, especially if we don't include `@babel/preset-env` in the example. Addressed in d1b18ff.

Sidenote: Brace expansion is first [introduced](https://en.wikipedia.org/wiki/C_shell) in `csh` and then copied to other Unix shells.